src/main/resources/test_files/runtime-spark/DataFrames/ForClauseType5.jq

```diff
@@ -0,0 +1,3 @@
+(:JIQS: ShouldCrash; ErrorCode="XPTY0004"; ErrorMetadata="LINE:2:COLUMN:20:" :)
+for $x as string in parallelize((1, 2))
+return $x
```

Comment:
This file should be named as 'ForClauseTypeCheckError[number].jq'

Reply:
DONE